app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+# Post model
+class Post < ApplicationRecord
+  has_many :comments
+  belongs_to :user
+
+  def update_posts_counter
+    User.find_by(id: user_id).increment(:postsCounter).save
+  end
```

Comment:
- [ ] Optional:
- Suggestions: You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private.
- EXAMPLE:
```ruby
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```